### DISCUSSION TITLE
How can I create a pie chart showing the lines between the bar gaps?

### DISCUSSION THREAD

**Discussion by rebecalrocha** [recharts/recharts#3081] [Q&A]
I would like to create a semi circle chart showing both the bar information and the information between the bar gaps.

Today the chart looks [like this](https://i.stack.imgur.com/SIyMm.png), but I would like to have something [like this](https://i.stack.imgur.com/QGkRQ.png).

It's possible?

**Comment by ckifer** [comment]
Hi @rebecalrocha 

Do you mind sharing some code? I don't think its possible but just want to check.
